scripts/test_update_smoke.sh

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
+
+# Run smoke tests to test that updating between versions work.
+#
+# This is based on our update tests but doing some tweaks to ensure we
+# can run it on Forge (or any other PostgreSQL server that only permit
+# a single user and single database).
+#
+# In particular, we cannot create new roles and we cannot create new
+# databases.
+#
+# For this version, we focus on single-node update and smoke testing
+# cannot (necessarily) be done with multinode setups. Feel free to try
+# though.
+#
+# The following environment variables can be set: 
+# - UPDATE_FROM_TAG is the version to update from (optional).
+#
+# - UPDATE_TO_TAG is the version to update to (optional).
+#
+# - PGHOST is host to use for the connection (required).
+#
+# - PGPORT is the port to use for the connection (required).
+#
+# - PGDATABASE is the database to use for the connection (required).
+#
+# - PGUSER is the username to use for the connection (required).
+#
+# - PGPASSWORD is the password to use for the connection
+#   (optional). If not set, password from .pgpass will be used (if
+#   available).
+
+: ${PGUSER:?Please provide the user in PGUSER}
+: ${PGHOST:?Please provide the host in PGHOST}
+: ${PGPORT:?Please provide the port in PGPORT}
+: ${PGDATABASE:?Please provide the database in PGDATABASE}
```

Comment:
none of this is actually needed as psql has reasonable defaults for those and you are not doing anything with those values here as this is just normal psql behaviour which you rely on

Reply:
Removed, and also show connection information using `\conninfo` instead.